### PR TITLE
Update ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by sol695510**
### Description of the Change

Update the descriptions of fields to be filled in the Issue Template.

### Alternate Designs

N/A

### Why Should This Be In Core?

N/A

### Benefits

Provides clearer goals for fixing the issue.

### Possible Drawbacks

N/A

### Verification Process

N/A

### Applicable Issues

#1103

### CODE REVIEW

**Review by aldebaran30701 — CHANGES_REQUESTED** [review]
I'd maybe suggest writing it only once - like a section just above Expected Behaviour. This section could say
```
Strive to include measurable metrics that can be aimed for when describing behaviour.

e.g. action 'x' should take 5 seconds or less to finish, NOT action 'x' should finish faster.

e.g. action 'x' takes 4 clicks, NOT action 'x' multiple clicks to be accomplished.
```

Let me know your thoughts

**Comment by sol695510** [comment]
> I'd maybe suggest writing it only once - like a section just above Expected Behaviour. This section could say
> 
> ```
> Strive to include measurable metrics that can be aimed for when describing behaviour.
> 
> e.g. action 'x' should take 5 seconds or less to finish, NOT action 'x' should finish faster.
> 
> e.g. action 'x' takes 4 clicks, NOT action 'x' multiple clicks to be accomplished.
> ```
> 
> Let me know your thoughts

The "strive" part can be just written once, but the "e.g." parts are not all the same.

**Comment by aldebaran30701** [comment]
>The "strive" part can be just written once, but the "e.g." parts are not all the same.

Understood, that's why I worked in an example of each.

I'm thinking with the new changes made, it should only include one example for each heading. We don't want to make it a very cluttered PR template, and as such only one line should be suffice to get the point across.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -43,12 +43,17 @@ https://github.com/constellation-app/constellation/blob/master/CODE_OF_CONDUCT.m
 
 3.  \[and so on…\]
 
+### Behaviour
+
+Try to include measurable metrics that can be aimed for (if applicable to the issue),
+**e.g.** If its a time issue then "should take 5 seconds or less to finish"
+or if its a workflow issue "should be accomplished in 2 or less clicks".
```
Comment (antares1470) [inline ISSUE_TEMPLATE.md:50]: I would comment this out (i.e. surround it in \).

Also "it's a time" and "it's a workflow"

**Review by antares1470 — CHANGES_REQUESTED** [review]

**Review by antares1470 — APPROVED** [review]

**Review by arcturus2 — APPROVED** [review]

**Pull request merged**